src/nidcpower/metadata/attributes_addon.py

```diff
@@ -4,6 +4,19 @@
 # We are not code genning attributes that have been marked as obsolete prior to the initial
 # Python API bindings release
 
+attributes_converters = {
+    1150046: { 'attribute_class': 'AttributeViReal64TimeDeltaSeconds', 
```

Comment:
This might be a dumb question, but why do you need to specify both the class and the converter? Doesn't the class suffice?

Reply:
The `'attribute_class'` specifies both the data type to convert to as well as the units that that value should be in. `'python_api_converter_type'` says what the datatype in the API is. This is what is used in documentation.

Reply:
Hmmm... seems redundant to have to specify both since the python type can be inferred from the attribute class. But since it's needed at codegen time, I don't have a better solution in mind. Something to keep in the back of our minds.

How come it's `'python_api_converter_type'` and not `'python_type'`?

Reply:
Needed something different when we could take either a datetime object or the underlying type. Now that we only accept datetime, `'python_type'` works.